packages/browser/src/browser/__tests__/integration.test.ts

```diff
@@ -878,6 +878,90 @@ describe('addDestinationMiddleware', () => {
     })
   })
 
+  it('drops events if  next is never called', async () => {
```

Comment:
These are not new, but there didn't seem to be any explicit test coverage for these scenarios.